content/en/integrations/guide/aws-manual-setup.md

```diff
@@ -79,6 +79,9 @@ Create an IAM role for Datadog to use the permissions defined in the IAM policy.
 {{< site-region region="ap1" >}}
 10. Enter `417141415827` as the `Account ID`. This is Datadog's account ID, and grants Datadog access to your AWS data.
 {{< /site-region >}}
+{{< site-region region="us1-fed" >}}
+10. If the AWS account you want to integrate is a GovCloud account, enter `065115117704` as the `Account ID`, otherwise enter `392588925713`. This is Datadog's account ID, and grants Datadog access to your AWS data.
```

Comment:
```suggestion
10. Enter `065115117704` as the `Account ID`, otherwise enter `392588925713`. This is Datadog's account ID, and grants Datadog access to your AWS data.
```

Reply:
Don't fully follow this suggestion, we would want the customer to change the account ID depending on which type of aws account theyre integrating

Reply:
I was thinking since the information is using the `gov/us1-fed` shortcode, this information will only show up if the user has US1-FED selected, so it'd be redundant. But I just realized that it's referring to the AWS account as a GovCloud account and not Datadog, so disregard my suggestion!